pkg/controller/sparkapplication/job.go

```diff
@@ -30,62 +35,96 @@ import (
 	"github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/config"
 )
 
+const (
+	sparkSubmitPodMemory = "1000Mi"
```

Comment:
The default resource requests are too high for a simple submit job.

Reply:
reduced to 250Mi/250m. right now, i pass the same defaults for both requests and limits. let me know if you think we should set a higher limit

Reply:
The default is still too high IMO. Let's set a low request but a higher limit.